### PR TITLE
42 blog titles

### PR DESCRIPTION
**Pull request by jspaaks**
This PR adds newly updated blog titles and updates hyperlinks accoringly for blog webassembly, react, web-worker.

I used `npm install -g markdown-link-check` to check the validity of the links.

### CODE REVIEW

File: webassembly/README.md [hunk 1]
```diff
@@ -283,7 +283,7 @@ In upcoming blogs we will cover:
 - [plotting with vega.js: how to visualize data from the algorithm]()
```
Comment (sverhoeven) [inline webassembly/README.md]: Can the in the bullet list also be set?

File: react/README.md [hunk 2]
```diff
@@ -38,7 +38,7 @@ We will keep the html code very minimal. The code will contain three essential e
     <script type="text/babel" src="app.js"></script>
   ```
 
-**Note:** We use the same `newtonraphson.js` and `newtonraphson.wasm` files as in the [previous post](TODO). Make sure you download these files from [here](TODO).
+**Note:** We use the same `newtonraphson.js` and `newtonraphson.wasm` files as in the [first post](../webassembly/README.md) of this series. Make sure you download these files from [here](../webassembly).
```
Comment (sverhoeven) [inline react/README.md]: Correct download link
```suggestion
**Note:** We use the same `newtonraphson.js` and `newtonraphson.wasm` files as in the [first post](../webassembly/README.md) of this series. Make sure you download [newtonraphson.js](https://github.com/NLESC-JCER/run-cpp-on-web/blob/master/react/newtonraphson.js) and [newtonraphson.wasm](https://github.com/NLESC-JCER/run-cpp-on-web/blob/master/react/newtonraphson.wasm) files from GitHub.
```

**Review by sverhoeven — CHANGES_REQUESTED** [review]
Changes look OK, but found a couple of missing entries.

**Comment by jspaaks** [comment]
Argh effing line endings in web-worker/README

File: webassembly/README.md (hunk 1 above)
Comment (jspaaks) [reply webassembly/README.md]: done

File: web-worker/README.md [hunk 3]
```diff
@@ -1,133 +1,139 @@
...
+_How to prevent blocking when running long running tasks in C++?_
+
+In this blogpost, we will use web workers to solve this problem by offloading tasks to another thread.
+
+## Long-running tasks with web worker
+
+Let's have a look at the code we ended up with in the last blog. When loading the page, the WebAssembly code is
+executed, after which the page can finish rendering. Because the WebAssembly code was very quick, this was fine. For
+this blog, we assume we have a longer running task. We create such a task artificially, by adding a few seconds of
+`sleep` in the [C++ code](newtonraphson.cpp). Like in the previous post, we compile the C++ code to create
```
Comment (sverhoeven) [inline web-worker/README.md]:
```suggestion
`sleep` in the [C++ code](https://github.com/NLESC-JCER/run-cpp-on-web/blob/master/web-worker/newtonraphson.cpp). Like in the previous post, we compile the C++ code to create
```

**Review by sverhoeven — APPROVED** [review]
Looks OK and link checker agrees.

One small suggestion

**Pull request merged**